bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/DatetimePickerConfig.java

```diff
@@ -245,6 +245,18 @@ public DatetimePickerConfig withEnabledDates(Date[] enabledDates) {
     }
 
     /**
+     * Set Option
+     *
+     * @param key jquery key
+     * @param value new value for the key
+     * @return config instance
+     */
+    public <T> DatetimePickerConfig withOption(IKey<T> key, T value) {
```

Comment:
This is not the right place/way.
This is basically making `#put()` public instead of protected. 
We intentionally made it protected so that you list the available options as setter methods with fluent API.
With this method it will be possible to set non-sense options.